app/lib/actions/katello/capsule_content/sync.rb

```diff
@@ -33,8 +33,15 @@ def plan(smart_proxy, options = {})
 
           smart_proxy_helper = ::Katello::SmartProxyHelper.new(smart_proxy)
           repositories = smart_proxy_helper.repos_available_to_capsule(environment, content_view, repository)
-          smart_proxy.ping_pulp3 if repositories.any? { |repo| smart_proxy.pulp3_support?(repo) }
-          smart_proxy.ping_pulp if repositories.any? { |repo| !smart_proxy.pulp3_support?(repo) }
+
+          smart_proxy.ping_pulp3 if repositories.any? do |repo|
+            smart_proxy.pulp3_support?(repo) &&
+            !smart_proxy.pulp2_preferred_for_type?(repo.content_type.to_sym)
+          end
+          smart_proxy.ping_pulp if repositories.any? do |repo|
+            !smart_proxy.pulp3_support?(repo) ||
+            smart_proxy.pulp2_preferred_for_type?(repo.content_type.to_sym)
+          end
```

Comment:
i would just remove this for capsule syncing, as we won't ever have a capsule with mixed types, and this setting only applies to the 'master' pulp server.